### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by stevenstetzler** [spsatuva/spsatuva.github.io#13]
We need to have documentation explaining how the website works for any incoming webmaster. We need to compile a list of things that need explaining. So far:
- How to make events
    - Create new file with year-month-day-title.md structure
    - Including lazy loaded images into posts
- Adding images to the site
    - Current image storage structure:
        - assets/img/posts for posts
        - assets/img/exec for exec board images
        - assets/img/featured-img for hero images on pages
    - Converting all images to jpg
        - ImageMagick `convert` and `mogrify` commands
    - Using `gulp img` to re-scale images so they are responsive
- Altering CSS
    - Current CSS structure:
        - Add new CSS to _sass/style/ and add an import statement in assets/css/main.scss
    - Basic information about CSS - classes and ids (they can figure out the rest from there)
        - Links to CSS help
- Brief overview of Jekyll
    - Discuss how things get piped through templates to make the final site
    - Links to docs
- Altering the exec board
    - _data/exec-board.yml structure

**Comment by stevenstetzler** [comment]
Note: We can use the [wiki](https://github.com/spsatuva/spsatuva.github.io/wiki) on the github page for this!

**Comment by stevenstetzler** [comment]
Lot's done: <https://spsatuva.github.io/docs>

To-Do:
- How to get the site running
- Altering research page
- Altering phys 1910 page
- Altering outreach guide page
- Jekyll overview and how to get the site running
- Slim down sleek documentation

**Comment by stevenstetzler** [comment]
@efrain99: wanted to keep you in the loop that @ColeMiles and I are making changes to the site documentation available at [https://spsatuva.github.io/docs](https://spsatuva.github.io/docs) so that the site isn't horribly confusing to navigate and learn about. I think our goal for the documentation should be that each piece of the website should be described in enough detail that if a webmaster wants to make a change to a page, they can just reference one page of the docs to learn what they need to do. We're updating the docs in a new branch called `docs-update-steven`. Once we've made satisfactory progress, we'll do a pull into the `master` branch but if you want to see the changes locally you can do (I'm pretty sure, see [here](https://stackoverflow.com/questions/9537392/git-fetch-remote-branch)):
```
git checkout --track origin/docs-update-steven
```
followed by
```
bundle exec jekyll serve
```
to build the site locally and view the docs at [localhost:4000/docs](http://localhost:4000/docs). `git checkout master` will get you back to the master branch,

A new to-do list for the docs that I've sketched out is the following:
- Describe Events page
- Describe Courses page
- Describe Grad School page
- Describe PHYS 1910 - About page
- Describe PHYS 1910 - Lectures page
- Describe Outreach - About page
- Describe Outreach - Guides page
- Describe Contact page
- Add basic site build instructions page
- Update README.md

We can also add items to this to-do list and alter existing documentation pages with your input on what may have been confusing for you since you've had to learn how to manage the page from scratch.

**Comment by stevenstetzler** [comment]
Resolved with #16
